transfer/build.gradle

```diff
@@ -0,0 +1,34 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 26
+
+
+
+    defaultConfig {
+        minSdkVersion 19
```

Comment:
because this module is part of the entire repo you can and should use  'minSdkVersion = rootProject.ext.minSdkVersion'